staging/src/k8s.io/kubectl/pkg/cmd/util/factory.go

```diff
@@ -35,9 +35,6 @@ import (
 // it uses the custom version of the function. Rather than try to enumerate everything that someone would want to override
 // we split the factory into rings, where each ring can depend on methods in an earlier ring, but cannot depend
 // upon peer methods in its own ring.
-// TODO: make the functions interfaces
-// TODO: pass the various interfaces on the factory directly into the command constructors (so the
```

Comment:
This TODO is still valid. Factory should be split into more granular pieces and until that time, this TODO should stay.
/hold